apps/voting/contracts/Voting.sol

```diff
@@ -176,7 +186,7 @@ contract Voting is IForwarder, AragonApp {
     * @param _evmScript Start vote with script
     */
     function forward(bytes _evmScript) public {
-        require(canForward(msg.sender, _evmScript));
+        require(canForward(msg.sender, _evmScript), ERROR_CAN_NOT_FORWARD);
         _newVote(_evmScript, "", true, true);
     }
 
```

Comment:
Can't comment far enough down, but there should be a message for the `require(vote_.totalVoters > 0)` later.

Reply:
Oh, I missed that one when merging `master`, thanks!